build_runner/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.12-prerelease
```

Comment:
is "prerelease" better than "dev" in some way?

Reply:
I think so yes, dev released in theory could be published, this more clearly indicates that it's not imo.